nidirect_breadcrumbs/nidirect_breadcrumbs.services.yml

```diff
@@ -95,3 +93,6 @@ parameters:
     - user.reset.login
     - nidirect_contacts.default
     - nidirect_contacts.letter
+    - entity.webform.canonical
+    - entity.webform.confirmation
+    - entity.node.webform.confirmation
```

Comment:
Webforms will be caught by the null breadcrumb builder and will get an empty breadcrumb trail unless they have a theme assigned, in which case they will have already been picked up by NodeThemesBreadcrumb